MVP/sample_job.yaml

```diff
@@ -0,0 +1,29 @@
+- job-template:
+    name: '{name}-{component}'
+    defaults: defaults-build
+    #node: 'jslave-of-your-own'
+    properties:
+        - ownership:
+            owner: lilu
+            co-owners:
+                - lilu
+                - lilu
+    triggers:
+        - ci-trigger:
+            jms-selector: "name = '{component}' AND CI_TYPE = 'brew-tag' AND (tag LIKE 'RHEL-6._-candidate' OR tag LIKE 'rhel-7._-candidate')"
+    builders:
+        - shell: |
+            set -x
+
+            # Here goes your shell commands for submitting beaker job here, and save the job ID into env var BKR_JOBID
+            # E.g. `bkr workflow-tomorrow --distro=$distro --task-file=$taskfile $other_args`
+
+            BKR_JOBID="J:1000000"
+            bkr job-watch $BKR_JOBID
+            bkr job-results --format junit-xml $BKR_JOBID > $WORKSPACE/$BKR_JOBID.xml
```

Comment:
Yes, we have a post-build action (in template yaml) to parse the junit file.
Here the `bkr job-results` command is a step for retrieving test results from beaker and write into a junit file for post-build action.